File: BOJ/3034.cpp
#include <cstdio>
#include <algorithm>
#include <math.h>
using namespace std;
int n, h, w;
int leng[51];
int main(){
	scanf("%d%d%d", &n, &h, &w);
	int Max = sqrt((h*h) + (w*w));
	for (int i = 1; i <= n; i++) {
		scanf("%d", &leng[i]);
		if (Max >= leng[i]) printf("DA\n");
		else printf("NE\n");
	}
	return 0;
}

File: BOJ(Java 공부용)/3034.java
import java.io.*;
import java.util.Arrays;
import java.util.StringTokenizer;

public class Main {

    static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
    static int[] gox = {1, -1, 0, 0};
    static int[] goy = {0, 0, -1, 1};
    static int ans = 0;

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int n = Integer.parseInt(st.nextToken()), h = Integer.parseInt(st.nextToken()), w = Integer.parseInt(st.nextToken());
        int max = (int) Math.sqrt(h * h + w * w);
        int[] arr = new int[n];
        for (int i = 0; i < n; i++) {
            arr[i] = Integer.parseInt(br.readLine());
        }
        Arrays.stream(arr)
                .forEach(value -> {
                    if (max >= value) System.out.println("DA");
                    else System.out.println("NE");
                });
    }
}

class P {
    int num;

    P(int num) {
        this.num = num;
    }

    public int getNum() {
        return num;
    }

    public void setNum(int num) {
        this.num = num;
    }
}